docker-images/jenkins/Dockerfile

```diff
@@ -58,5 +49,3 @@ RUN unzip /wso2am-micro-gw-toolkit-3.0.1.zip
 
 RUN ls /wso2am-micro-gw-toolkit-3.0.1/bin
 ENV PATH="${PATH}:/wso2am-micro-gw-toolkit-3.0.1/bin"
```

Comment:
@Aaquiff shall we have one Dockerfile `ENV` command and set all environment variables in that? Please see the example at [1].

[1]: [Single `ENV` command](https://github.com/wso2/docker-apim/blob/v2.6.0.7/dockerfiles/ubuntu/apim/Dockerfile#L88)